src/main/models/Custom_Set1.java

```diff
@@ -0,0 +1,53 @@
+package main.models;
```

Comment:
Custom_Set1 and Custom_Set2 do not seem to be models and probably need to be moved to a separate package named util (e.g., java.util.SortedSet in JDK)